packages/page-democracy/src/Overview/Referendum.tsx

```diff
@@ -143,6 +145,18 @@ function Referendum ({ className = '', value: { allAye, allNay, image, imageHash
           votes={allNay}
         />
       </td>
+      <td className='class mini media--1000'>
+
+        <CardSummary
+          progress={{
```

Comment:
I don't believe we should be using a `CardSummary` here at all. The `<Progress ... />` component itself does all the work. (The summary is only a wrapper around it)